etna/analysis/feature_selection/mrmr_selection.py

```diff
@@ -30,11 +30,22 @@ class AggregationMode(str, Enum):
 }
 
 
+def is_relevant(relevance, feature, atol):
+    """Return True if feature is relevant, else return False."""
+    return not relevance.loc[feature] == 0
+
+
+def is_not_relevant(relevance, feature, atol):
```

Comment:
Do you really need this functions? You could just add this logic into filter explicitly without functions